### PR TITLE
dhruv

### PR DESCRIPTION
**Pull request by techiesdhruv**

### CODE REVIEW

File: index3.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Repo created from Local System</title>
 </head>
 <body>
-    
+    this is dhruv
```
Comment (shrikant2503) [inline index3.html:9]: but i am shrikant bro

**Review by shrikant2503 — CHANGES_REQUESTED** [review]
Add My Name

File: index3.html (hunk 1 above)
Comment (techiesdhruv) [reply index3.html:9]: ok

**Review by shrikant2503 — COMMENTED** [review]
OKay...................Thanks

**Review by techiesdhruv — COMMENTED** [review]
d

**Pull request merged**